apps/meteor/client/components/PageEmptyPrivateApps.tsx

```diff
@@ -0,0 +1,23 @@
+import { States, StatesIcon, StatesTitle, StatesSubtitle, StatesSuggestion } from '@rocket.chat/fuselage';
+import { Link } from '@rocket.chat/layout';
+import { useTranslation } from '@rocket.chat/ui-contexts';
+import React from 'react';
+
+function PageEmptyPrivateApps() {
```

Comment:
update this component to 'const PageEmptyPrivateApps = () =>`, we use the last versions in terms of ECMAScript, don't worry about compatibility because we use babel.js...